adapters/iqx/iqx.go

```diff
@@ -0,0 +1,161 @@
+package iqx
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"text/template"
+
+	"github.com/prebid/openrtb/v19/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/macros"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type bidType struct {
+	Type string `json:"type"`
+}
+
+type bidExt struct {
+	Prebid bidType `json:"prebid"`
+}
```

Comment:
scope of `Prebid` field is limited to package `iqx` . Therefore can make this field private.

```
type bidExt struct {
	prebid bidType `json:"prebid"`  // make prebid private by starting with a lower case char
}
```

Reply:
@onkarvhanumante thank you for the review!
sorry, but looks like I cant unmarshal bid.Ext after this change